docs/src/app/components/pages/components/popover.jsx

```diff
@@ -87,6 +87,16 @@ let PopoverPage = React.createClass({
       <ComponentDoc
         name="Popover"
         componentInfo={componentInfo}>
+        
+        <Paper style = {{marginBottom: '22px'}}>
+          <CodeBlock>
+          {
+            '//Import statement:\nimport Popover from \'material-ui/lib/popover\';\n\n' +
```

Comment:
I thought it was weird too, but I followed the context:

* https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/avatars.jsx#L78
* https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/snackbar.jsx#L111
* https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/dialog.jsx#L197
* Etc...